clang/test/CodeGen/X86/movrs-avx10.2-512-builtins-error-32.c

```diff
@@ -0,0 +1,50 @@
+// RUN: %clang_cc1 -ffreestanding %s -Wno-implicit-function-declaration -triple=i386-- -target-feature +movrs -target-feature +avx10.2-512 -emit-llvm -verify
```

Comment:
Since we don't care about the actual output, can we remove "-emit-llvm" here to avoid writing to the local disc (which may be write-protected)?
See e.g. b7730a23ef
```[test] Avoid writing to a potentially write-protected dir (#102073)```
which solves a similar issue.

Reply:
Thanks for the report, fixed by https://github.com/llvm/llvm-project/commit/c2d2b3b80866633b4db65c3841c40c16815267f3

Reply:
Thanks!